xbmc/FileItem.cpp

```diff
@@ -124,6 +124,10 @@ CFileItem::CFileItem(const CEpgInfoTagPtr& tag)
   Initialize();
 
   m_bIsFolder = false;
+  if (tag->HasPVRChannel())
+  {
+    m_pvrChannelInfoTag = tag->ChannelTag();
```

Comment:
I wasn't sure about this. Who could guide here how to solve this correctly? If the channel is not set, lots of checks need to be changed to ensure that the channel information is still printed (we are actually playing a channel here, just not live).